charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -59,6 +59,13 @@ data:
   {{- end }}
   {{ .Values.server.coordinatorExtraConfig | indent 4 }}
 
+{{- if .Values.accessControl }}{{- if .Values.accessControl.type }}
+  access-control.properties: |
+    access-control.name=file
+    security.refresh-period={{ .Values.accessControl.refreshPeriod | default "1s" }}
```

Comment:
1s seems overkill. With a moderately large access control config you'd end wasting lot of CPU time just reloading the config. A better default is to disable refresh. If we want to keep it enabled it should be something like 60s maybe?

However on principle I think the helm chart should follow default values that Trino has - some benefits are consistency and least amount of surprise for people moving to this chart from some other deployment method.

cc: @electrum WDYT about making sure chart defaults match Trino defaults? If there's good justification to have a different default in Chart maybe it makes more sense to change defaults directly in Trino to benefit a wider audience?

Reply:
I disabled the refresh period unless `.Values.accessControl.refreshPeriod` exists and set the default to 60s (I'm not sure what the default is).

Reply:
There is no default in Trino. If it's unset then refresh is disabled, if it's set to any other value refresh is enabled (min is 1ms). It's simpler than having both a "enable" and "duration" toggle.